### DISCUSSION TITLE
problematic eslint setup in examples

### DISCUSSION THREAD

**Discussion by o-az** [vercel/turborepo#1182] [Help] [answered]
In the kitchen-sink example, eslint package is named `eslint-config-custom` and in the other packages where eslint config is imported/extended, the `.eslintrc.js` file is extending `"custom"`.

How does eslint know that `extends: ["custom"]` means extend the eslint config from `packages/eslint-config-custom`?

https://github.com/vercel/turborepo/blob/cdd6ebb91ac3d0ccf23c2c09e00092b28834e987/examples/kitchen-sink/packages/ui/.eslintrc.js#L3

https://github.com/vercel/turborepo/blob/cdd6ebb91ac3d0ccf23c2c09e00092b28834e987/examples/kitchen-sink/packages/eslint-config-custom/package.json#L2

I've been having issues with eslint because it cannot recognize what `custom` is.

**Comment by AbdallahAbis** [comment, marked as answer]
your eslint config should be working I believe. `eslint-config-custom` actually can be extended as `custom`. check out eslint docs on sharable config https://eslint.org/docs/developer-guide/shareable-configs
